### DISCUSSION TITLE
Integrate slideshow view for upload cells

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#9101] [Feature Requests & Ideas]
Currently, it's fairly difficult to preview selected Uploads in an upload field with hasMany: true. It would be great if Payload could integrate some sort of media viewer to make viewing the media easier.

**Comment by elliott-w** [comment]
Just adding on to this:

Would be great to have an admin option to change the view type when **selecting** the files, either:
- Table/list view (as it is currently)
- Gallery view (think how media select works in wordpress where the images are displayed in a grid)
